cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/InstancePricingDTO.java

```diff
@@ -15,19 +16,25 @@
 @ApiModel( description = "Defines pricing of disk usage of a VM" )
 public class InstancePricingDTO extends CloudPricingDTO
 {
-    @ApiModelProperty( "Descriptive name of the pricing" )
+    @NotNull
+    @Valid
+    private UrlEntityDTO instanceType;
+
+    @ApiModelProperty( value = "Descriptive name of the pricing", hidden = true )
+    @Deprecated
     @Size( max = 125, message = INVALID_SIZE )
     private String name;
 
-    @ApiModelProperty( "RAM quantity" )
-    @NotNull
+    @ApiModelProperty( value = "RAM quantity", hidden = true )
+    @Deprecated
     private Integer ram;
 
-    @ApiModelProperty( "CPU Cores" )
-    @NotNull
+    @ApiModelProperty( value = "CPU Cores", hidden = true )
+    @Deprecated
     private BigDecimal cpu;
 
-    @ApiModelProperty( "Root disk space" )
+    @ApiModelProperty( value = "Root disk space", hidden = true )
+    @Deprecated
     private Integer diskSpace;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)